apps/test/solutions/applab/ec_design.js

```diff
@@ -279,19 +279,69 @@ module.exports = {
       runBeforeClick: function (assert) {
         $("#designModeButton").click();
         testUtils.dragToVisualization('BUTTON', 10, 20);
+
+        var buttonElement = document.getElementById('design_button1');
+        var $designModeViz = $('#designModeViz');
+
+        // test initial position of button
         assertPropertyRowValue(0, 'id', 'button1', assert);
         assertPropertyRowValue(2, 'width (px)', 80, assert);
         assertPropertyRowValue(3, 'height (px)', 30, assert);
         assertPropertyRowValue(4, 'x position (px)', 10, assert);
         assertPropertyRowValue(5, 'y position (px)', 20, assert);
-
-        dragElement(document.getElementById('design_button1'), 20, 0);
-        assertPropertyRowValue(4, 'x position (px)', 30, assert);
-        assertPropertyRowValue(5, 'y position (px)', 20, assert);
-
-        dragElement(document.getElementById('design_button1'), 0, 20);
-        assertPropertyRowValue(4, 'x position (px)', 30, assert);
-        assertPropertyRowValue(5, 'y position (px)', 40, assert);
+        
+        // test dragging within visualization
+        // test button dragged 20 pixels in X direction
+        //dragElement(buttonElement, 20, 0);
+        //assertPropertyRowValue(4, 'x position (px)', 30, assert);
+        //assertPropertyRowValue(5, 'y position (px)', 20, assert);
+
+        // test button dragged 20 pixels in Y direction
+        //dragElement(buttonElement, 0, 20);
+        //assertPropertyRowValue(4, 'x position (px)', 30, assert);
+        //assertPropertyRowValue(5, 'y position (px)', 40, assert);
```

Comment:
What is the story with the commented-out sections of these tests? Are you are running into unexpected errors when you uncomment them?